rs/core/corpora_cli/src/main.rs

```diff
@@ -18,15 +20,16 @@ struct Cli {
 
 fn main() {
     let cli = Cli::parse();
+    let ctx = Context::new("http://app:8877");
```

Comment:
I think we just call it with no arguments and let it run wild from there .. with the full bespoke context ...